package.json

```diff
@@ -76,5 +78,10 @@
       "html",
       "lcov"
     ]
+  },
+  "dependencies": {
+    "axios": "^0.16.0 || ^0.17.0 || ^0.18.0",
+    "deep-freeze-strict": "^1.1.0",
+    "js-cookie": "^2.1.4"
```

Comment:
This bit gets added by `yarn peers`. Not sure of a good way to avoid it showing up in diffs, but just back it out for now.

Reply:
10-4. Why do we do it this way, using peers instead of regular dependencies for these things?

Reply:
That's the way you're supposed to do it when the dependencies come from the installing project. 🤷‍♂